packages/cosmic-swingset/lib/block-manager.js

```diff
@@ -172,6 +172,10 @@ export default function makeBlockManager({
         // Advance our saved state variables.
         savedActions = currentActions;
         computedHeight = action.blockHeight;
+        if (action.blockHeight === 0) {
```

Comment:
I don't know how this relates to the new `agoric` subcommands.

In reading through the block manager again, I noticed that the name `savedActions` implies that subsequent changes to `currentActions` won't affect the saved ones, but of course we're just starting a separate reference to the same mutable Array. Whatever it's doing, it seems to work, but it's confusing. And this change clears `currentActions`, but also clears `savedActions` as a side-effect.

If the intention is to prevent a restart from a state where we've recorded the genesis block but nothing else, maybe this should set `savedActions` instead of / in addition to `currentActions`?

Reply:
The new commands tickle the debug statement that lets you know when uncommitted txes appear before the `BEGIN_BLOCK` of `blockHeight = 1`.  This statement is to enforce the Cosmos constraint that `blockHeight = 0` is special, and not committed in the same way.